### ISSUE TITLE
Adding OG File Name Variable for Output

### ISSUE THREAD

**Issue by stark4n6** [EricZimmerman/SQLECmd#54]
** SQLECmd version **
1.0.0.0

**Is your feature request related to a problem? Please describe.**
Processing multiple similar formatted files that use the same Map cause indistinguishable resulting CSVs.

**Describe the solution you'd like**
When running with seek across Maps, would be great to have a potential variable to the "BaseFileName" field that would add the original file name that is processed to the final CSV filename.

**Describe alternatives you've considered**
N/A

**Additional context**
See screenshot below, the original DB files on the right and the resulting files on the left. I have to manually open each CSV to figure out which correlates to which original DB.

![Screenshot 2022-05-13 155328](https://user-images.githubusercontent.com/48143894/168380162-c509d3fb-9daf-4b20-9718-ed1413dfcc7f.png)

**Comment by AndrewRathbun** [comment]
So maybe `%f` which would put the filename of the DB being parsed into the BaseFilename, kinda like how KAPE works with `%m` and `%d` with Hostname and DateTime?

**Comment by EricZimmerman** [comment]
doesnt what is on the screen during the run help?

**Comment by EricZimmerman** [comment]
does the csv itself contain a source file field?

**Comment by AndrewRathbun** [comment]
> does the csv itself contain a source file field?

Yes it does.

**Comment by stark4n6** [comment]
> does the csv itself contain a source file field?

yeah it does, maybe I need to refactor the queries to see if I can tie the databases together somehow more easily.
